synth.py

```diff
@@ -20,31 +20,43 @@
 import logging
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 version = "v3"
-library = gapic.node_library(
-  "monitoring", version, config_path="/google/monitoring/artman_monitoring.yaml"
+library = gapic.typescript_library(
+    "monitoring",
+    version,
+    generator_args={
+        "grpc-service-config": f"google/monitoring/{version}/monitoring_grpc_service_config.json",
+        "package-name": f"@google-cloud/monitoring",
+        "main-service": f"monitoring"
+    },
+    proto_path=f'/google/monitoring/{version}',
+    extra_proto_files=['google/cloud/common_resources.proto']
 )
-s.copy(library, excludes=["src/index.js", "README.md", "package.json"])
+
+s.copy(library, excludes=["src/index.ts", "README.md", "package.json"])
 
 common_templates = gcp.CommonTemplates()
-templates = common_templates.node_library()
+templates = common_templates.node_library(source_location='build/src')
 s.copy(templates)
 
-# [START fix-dead-link]
-s.replace('**/doc/google/protobuf/doc_timestamp.js',
-        'https:\/\/cloud\.google\.com[\s\*]*http:\/\/(.*)[\s\*]*\)',
-        r"https://\1)")
+s.replace('src/v3/*.ts',
```

Comment:
I'd like to block on fixing these in the upstream protos

Reply:
Agree. I will send CL to fix.

Reply:
Update the broken link with correct prefix in upstream proto
https://critique-ng.corp.google.com/cl/295230066
Git rid of synth hack here